### PR TITLE
search: enables tc keyword

### PR DESCRIPTION
**Pull request by Panos512**
This is needed for inspirehep/inspire-next#976 .

Signed-off-by: Panos Paparrigopoulos panos.paparrigopoulos@cern.ch

### CODE REVIEW

File: invenio_query_parser/contrib/spires/config.py [hunk 1]
```diff
@@ -174,13 +176,13 @@
...
-    'type-code': 'collection',
-    'scl': 'collection',
-    'ps': 'collection',
+    'collection': 'tc',
+    'tc': 'tc',
+    'ty': 'tc',
+    'type': 'tc',
+    'type-code': 'tc',
+    'scl': 'tc',
+    'ps': 'tc',
```
Comment (kaplun) [inline invenio_query_parser/contrib/spires/config.py:183]: Uh? Why just replacing `collection` with `tc` is helping this?

File: invenio_query_parser/contrib/spires/config.py [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
 # -*- coding: utf-8 -*-
 #
-# This file is part of Invenio-Query-Parser.
-# Copyright (C) 2014 CERN.
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
 #
-# Invenio-Query-Parser is free software; you can redistribute it and/or
-# modify it under the terms of the GNU General Public License as
+# INSPIRE is free software; you can redistribute it
```
Comment (kaplun) [inline invenio_query_parser/contrib/spires/config.py]: I think the copyright should still stay to `Invenio-Query-Parser`. In this way we can send upstream these changes.

Comment (Panos512) [inline invenio_query_parser/contrib/spires/config.py]: Woops. It needed update and I copied it from the wrong file

**Comment by jacquerie** [comment]
Ach, this needs another rebase now that we've merged #15.

**Pull request merged**